TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -2268,4 +2009,3 @@ def partitions( self ):
             main.log.exception( self.name + ": Uncaught exception!" )
             main.cleanup()
             main.exit()
-
```

Comment:
So I'm not seeing any changes to this file that you specifically made, I think you just had an old version. It looks like all these are just reversions of recent changes made to master?